src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((acc, curr) => {
+      const [key, value] = curr.split(':').map((str) => str.trim());
```

Comment:
The line `const [key, value] = curr.split(':').map((str) => str.trim());` assumes that `curr.split(':')` will always return exactly two elements. If `curr` does not contain a colon, this will result in `key` being assigned a value and `value` being `undefined`. Consider adding a check to ensure `curr.split(':')` results in exactly two elements before destructuring.